content/en/docs/reference/commands/deprecation.md

```diff
@@ -15,12 +15,13 @@ description: list of jx commands which have been deprecated
 |----------------|----------------|--------------|
 | jx create post | Feb 1 2020 |  |
 | jx create spring | Mar 1 2020 | jx create project |
+| jx create vault | Sep 1 2020 |  |
 | jx delete extension | Feb 1 2020 |  |
+| jx delete vault | Sep 1 2020 |  |
```

Comment:
@MarckK @deanesmith, so the reason the deprecation page was not getting updated (as discussed in one of the docs meeting) was because the website doc generation job was broken, now that it's fixed, it seems to work yay :tada: 
relevent PR: https://github.com/jenkins-x/jx/pull/7351 (@abayer helped in tracking the root cause of this issue :100:)

Reply:
well done